types/filecontracts.go

```diff
@@ -108,15 +108,23 @@ func (fcid FileContractID) StorageProofOutputID(proofStatus ProofStatus, i uint6
 	))
 }
 
+// PostTax returns the amount of currency remaining in a file contract payout
+// after tax.
+func PostTax(height BlockHeight, payout Currency) Currency {
+	// COMPATv0.4.0 - until the first 20,000 blocks have been archived, they
+	// will need to be handled in a special way.
+	if (height < 21e3 && build.Release == "standard") || (height < 10 && build.Release == "testing") {
+		return payout.Sub(payout.MulFloat(0.039).RoundDown(SiafundCount))
+	}
+	return payout.Sub(payout.MulTax().RoundDown(SiafundCount))
+}
```

Comment:
isn't this just `payout.Sub(Tax(height, payout))`? Is a separate function warranted? I find the name a bit confusing too.

Also, maybe `Tax` could be a method of `Currency`?